Dockerfile

```diff
@@ -43,6 +44,13 @@ RUN add-apt-repository -y ppa:ondrej/php && \
 RUN curl -sS https://getcomposer.org/installer | php && \
     mv composer.phar /usr/local/bin/composer
 
+# Install excimer
+RUN git clone https://github.com/wikimedia/mediawiki-php-excimer.git excimer && \
+    cd excimer/ && \
+    phpize && ./configure && make && make install && \
+    echo 'extension=excimer.so' > /etc/php/${phpv}/mods-available/excimer.ini && \
+    cd / && rm -R ./excimer
+
 # Install WP-CLI
```

Comment:
This is where we install the `excimer` package. We are building from source to get the latest version (`v1.1.1`) that works with PHP8.2. At the time of writing, `pecl` and `Ubuntu` repos didn't have the latest version, they had `v1.0.4`